<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StocksModelsWidget</class>
 <widget class="QWidget" name="StocksModelsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="stocksTableView"/>
     </item>
     <item>
      <widget class="QTableView" name="stocksLimitsTableView"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
